shell/components/ModalManager.vue

```diff
@@ -38,6 +38,8 @@ function registerBackgroundClosing(fn: Function) {
       :click-to-close="closeOnClickOutside"
       :width="modalWidth"
       :style="{ '--prompt-modal-width': modalWidth }"
+      :trigger-focus-trap="true"
+      tabindex="0"
```

Comment:
mate, I don't know if you can enable this like that... I'll need to run some tests on it. It's better that you get your stuff in first and then we (me) worry about the focus trap... 😛

Reply:
The e2e tests were failing without it unfortunately.